Shared/Stores/UserStore.swift

```diff
@@ -50,28 +50,22 @@ class UserStore {
 
     private func subscribeToUpdates() async {
 
-
-        var unfetchedUsers = ConnectionStore.shared.connections.compactMap { connection in
-            return connection.nonMeUser
+        var unfetchedUserIds = ConnectionStore.shared.connections.compactMap { connection in
+            return connection.nonMeUser?.objectId
         }
 
-        if let current = User.current() {
-            unfetchedUsers.append(current)
+        if let current = User.current()?.objectId {
+            unfetchedUserIds.append(current)
         }
 
-        await withTaskGroup(of: User?.self) { group in
-            for user in unfetchedUsers {
-                group.addTask {
-                    return try? await user.fetchInBackground() as? User
-                }
+        if let users = try? await User.localThenNetworkArrayQuery(where: unfetchedUserIds,
```

Comment:
I added this as an optimization for extensions. We should be able to access the group container instead of querying the network every time.